src/test/java/Tests/TestAgreement.java

```diff
@@ -0,0 +1,20 @@
+package Tests;
+
+
+import mainpack.module.Agreement;
+import mainpack.module.subject.AbstractSubject;
+import mainpack.module.subject.Apartment;
+import org.junit.jupiter.api.Assertions;
+import org.testng.annotations.Test;
+
+
+public class TestAgreement {
```

Comment:
Please add "Test" after class name that should be tested, in this case: "AgreementTest"